### PR TITLE
docs: Add order_enterprise: <number>

### PR DESCRIPTION
**Pull request by anithamuthu2022**
[DOC-92]

[DOC-92]: https://heartex.atlassian.net/browse/DOC-92?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by anithamuthu2022** [comment]
https://user-images.githubusercontent.com/107003313/203633086-82219f3d-3ba8-49f3-8240-1ddd31ef7c63.mov

@huguestennier I've added `order:enterprise: <number> in the pages so that I would be able to see the following hierarchy of ordering in the left nav on the ENT website only. 

Unfortunately, it is not showing up as expected in the local build. What is the possible change here to make sure the left nav in the ENT website has the following order of pages?

```GET STARTED
CONFIGURATION
IMPORT AND EXPORT
LABELING
QUALITY CONTROL
MACHINE LEARNING
INTEGRATION AND DEVELOPMENT
SECURITY AND PRIVACY
BILLING AND USAGE
INSTALL
ADVANCED DEVELOPMENT```

**Comment by huguestennier** [comment]
@anithamuthu2022 before you create a new branch, make sure you the original branch is up to date. It wasn’t working because it was missing the code. I made a commit to make sure you have the latest code, should work now.

**Comment by anithamuthu2022** [comment]
@huguestennier can you review/approve this PR so I can see the new left nav on the ENT website?

I still don't see it in. my local build.

**Comment by makseq** [comment]
opensource:
image

**Comment by makseq** [comment]
enterprise order is broken now: 
image

**Review by makseq — CHANGES_REQUESTED** [review]
see my 2 previous comments

**Comment by anithamuthu2022** [comment]
ENT order is addressed on the PR.
GET STARTED
CONFIGURATION
IMPORT AND EXPORT
LABELING
QUALITY CONTROL
MACHINE LEARNING
INTEGRATION AND DEVELOPMENT
SECURITY AND PRIVACY
BILLING AND USAGE
INSTALL

**Comment by anithamuthu2022** [comment]
OSS order on the PR=> expected order is:
Get Started
Install
Configuration
Import and Export
Labeling 
Machine Learning 
Integration and Development

**Comment by anithamuthu2022** [comment]
@huguestennier I've edited the `home.ejs` file to meet the following requirements for the landing page, but I see the following errors: 

1. Both OSS and ENT landing pages have all the tile cards for OSS and ENT on the same page. This is not the expected behavior, but it should display only the [order of pages explained in Slack](https://heartex.slack.com/archives/C03UN6A0CQN/p1669655427316309?thread_ts=1669237012.108109&cid=C03UN6A0CQN) and also should include the two main tiles on the landing page. 
Screen Shot 2022-11-28 at 10 52 48 AM

2. The `Getting Started` tile and `Overview` tile are missing from OSS and ENT's landing page when you run the local build. 

Screen Shot 2022-11-28 at 10 55 12 AM
Screen Shot 2022-11-28 at 10 55 05 AM

**Comment by anithamuthu2022** [comment]
https://user-images.githubusercontent.com/107003313/204841694-46d94b98-c373-4dc4-8a0b-83fb44bfd8f1.mov

Works perfectly on the landing page and left nav for ENT website.

**Comment by anithamuthu2022** [comment]
https://user-images.githubusercontent.com/107003313/204845065-2f14e119-97aa-4f1a-ae90-fbcd28215bbe.mov

Both ENT and OSS works perfectly on the landing page and left nav. 


https://user-images.githubusercontent.com/107003313/204845107-9c4a64a5-4ee9-40d1-8961-a469763996fc.mov

File: docs/themes/v2/layout/home.ejs [hunk 1]
```diff
@@ -204,12 +182,172 @@ meta_description: Get started with Label Studio by creating projects to label an
...
+</div>
+
+<div class="enterprise-only">
+  <div class="columns">
+    <div class="card-light">
+        <!-- All the Enterprise cards are listed here -->
+      <%- partial("component/heading", {text: "Getting Started", tag: "h2", customClass: "card-title" }) %>
+      <%- partial("component/link", { url: "/guide/install.html", label: "Learn how to install", customClass: "card-cta" }) %>
+      </div>
+      <div class="card-light">
```
Comment (huguestennier) [inline docs/themes/v2/layout/home.ejs:203]:
```suggestion
      <div class="card card-light">
```

File: docs/themes/v2/layout/home.ejs [hunk 2]
```diff
@@ -204,12 +182,172 @@ meta_description: Get started with Label Studio by creating projects to label an
...
+              <%- partial("component/link", { url: "/guide/install_enterprise.html", label: "Install and Deploy", size: "Full" }) %>
+            </li>
+          </ul>
+    </div>
+
+    <div class="card">
+          <%- partial("component/heading", {text: "Reference", tag: "h2", size: "XSmall", customClass: "card-title"}) %>
+          <ul class="card-links-list">
+            <li class="enterprise-only">
+              <%- partial("component/link", { url: "/app.heartex.com/docs/api", label: "API and SDKs", size: "Full" }) %>
```
Comment (huguestennier) [inline docs/themes/v2/layout/home.ejs:338]:
```suggestion
              <%- partial("component/link", { url: "https://app.heartex.com/docs/api", label: "API and SDKs", size: "Full" }) %>
```

**Comment by anithamuthu2022** [comment]
@huguestennier @makseq could you review/approve this PR. 

AFAIK, https://heartex.slack.com/archives/C03UN6A0CQN/p1669823891132399?thread_ts=1669652993.899809&cid=C03UN6A0CQN is not including this in PR.

**Review by huguestennier — APPROVED** [review]

**Pull request merged**